### ISSUE TITLE
modulestf-ls server crashed

### ISSUE THREAD

**Issue by AJCandfield** [modulestf/public#3]
## Description

Extension seems to be working initially but crashes after about 3 minutes with the following message:
```
The modulestf-ls server crashed 4 times in the last 3 minutes. The server will not be restarted.
```
This is the output from the VSCode tab:
```
Starting...
[Info  - 10:35:57] Language service initialization...
[Info  - 10:35:57] folder: infrastructure -> /home/alex/code/infrastructure
[Info  - 10:35:58] folder: daze -> /home/alex/code/daze

Thank you for installing modules.tf - a VSCode extension for Terraform users!

This is an alpha release that will stop working after the 31th of January 2022.
We will publish another release by that time. Stay tuned for more updates!

[Follow @modulestf](https://twitter.com/modulestf)

[Info  - 10:36:10] Connection to server got closed. Server will restart.
[Info  - 10:36:10] Language service initialization...
[Info  - 10:36:10] folder: infrastructure -> /home/alex/code/infrastructure
[Info  - 10:36:10] folder: daze -> /home/alex/code/daze

Thank you for installing modules.tf - a VSCode extension for Terraform users!

This is an alpha release that will stop working after the 31th of January 2022.
We will publish another release by that time. Stay tuned for more updates!

[Follow @modulestf](https://twitter.com/modulestf)

[Info  - 10:36:20] Connection to server got closed. Server will restart.
[Info  - 10:36:20] Language service initialization...
[Info  - 10:36:20] folder: infrastructure -> /home/alex/code/infrastructure
[Info  - 10:36:20] folder: daze -> /home/alex/code/daze

Thank you for installing modules.tf - a VSCode extension for Terraform users!

This is an alpha release that will stop working after the 31th of January 2022.
We will publish another release by that time. Stay tuned for more updates!

[Follow @modulestf](https://twitter.com/modulestf)

[Info  - 10:36:34] Connection to server got closed. Server will restart.
[Info  - 10:36:35] Language service initialization...
[Info  - 10:36:35] folder: infrastructure -> /home/alex/code/infrastructure
[Info  - 10:36:35] folder: daze -> /home/alex/code/daze

Thank you for installing modules.tf - a VSCode extension for Terraform users!

This is an alpha release that will stop working after the 31th of January 2022.
We will publish another release by that time. Stay tuned for more updates!

[Follow @modulestf](https://twitter.com/modulestf)

[Error - 10:36:45] Connection to server got closed. Server will not be restarted.
```

## Versions
VSCode: 1.65.2

## Reproduction
* Install the extension from the marketplace
* Wait 3 minutes

## Additional context
Using a Ubuntu 20.04 workstation. I have installed the Terraform extensions by Hashicorp and Anton Kulikov.

**Comment by antonbabenko** [comment]
Unfortunately, our homeland was invaded by Russia before we were able to release a new version of this extension.

I'm going to publish a new release with an updated date (no new features added) as soon as I have time (today or tomorrow).

Sorry for the inconvenience!

**Comment by AJCandfield** [comment]
No problem mate! Stay safe! 🇺🇦

**Comment by antonbabenko** [comment]
I've just published an updated version 0.0.8 to the [VSCode Marketplace](https://marketplace.visualstudio.com/items?itemName=betajob.modulestf).

Have a great day!
